gradle/libs.versions.toml

```
@@ -33,6 +34,8 @@ ktor-server-auth-jwt = { group = "io.ktor", name = "ktor-server-auth-jwt", versi
 ktor-serialization-kotlinx = { group = "io.ktor", name = "ktor-serialization-kotlinx-json-jvm", version.ref = "ktorVersion" }
 ktor-client-content-negotation = { group = "io.ktor", name = "ktor-client-content-negotiation", version.ref = "ktorVersion" }
 
+kartverket-ws-client = { group = "no.kartverket.matrikkelapi-v1", name = "matrikkelapi-v1-wsclient", version.ref = "wsClientVersion" }
```

Så ikke det i går, men lurer på om denne burde hete noe med matrikkel, f.eks. dette. Så er det lettere å se at det er matrikkel APIet det er snakk om.
```suggestion
matrikkelapi-ws-client = { group = "no.kartverket.matrikkelapi-v1", name = "matrikkelapi-v1-wsclient", version.ref = "wsClientVersion" }
```

---

Vil du at jeg skal endre navn på denne også? `wsClientVersion`

---

Kan gjerne det, så det henger sammen. Feks. `matrikkelWsClientVersion` eller noe?